### PR TITLE
Fix(learn): Rework Euler Problem 125

### PR DESCRIPTION
**Pull request by blumerDev**
Checklist:

 - [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org/).
 - [x] My pull request has a descriptive title (not a vague title like Update index.md)
 - [x] My pull request targets the main branch of freeCodeCamp.
 - [x] I have tested these changes either locally on my machine, or GitPod.

- Changed challenge to use argument in function.
- Added three additional tests.
- Tested on local fork.
- Added example solution
- Related to #40896

### CODE REVIEW

**Comment by gikf** [comment]
As a new contributor, we encourage you to read our [contributing guidelines](https://contribute.freecodecamp.org).

We expect our contributors to be aware of the process specific to this project. Following the guidelines religiously earns you the respect of fellow maintainers and saves everyone time.

Some examples of this are:

1. Do not edit files directly through GitHub – while you can, it's not a good idea.
2. Make sure you follow the PR checklist and not just tick things off; otherwise, we won't take you seriously.
3. Use the correct way to link issues in the description of the PR by updating the `XXXXXX`. Do not just add issue numbers everywhere and anywhere you feel like.
4. Keep mentions and review requests to a minimum. We understand you are excited about contributing, and our maintainers will get back to you as soon as they get a chance.
5. Do not work directly off your `main` branch - create a new branch for the changes you are working on.

We appreciate you taking the time to help us, and we hope to see more contributions from you.

Happy Contributing.

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-125-palindromic-sums.md [hunk 1]
```diff
@@ -12,31 +12,75 @@ The palindromic number 595 is interesting because it can be written as the sum o
...
 
-palindromicSums();
+palindromicSums(100000000);
 ```
 
 # --solutions--
 
 ```js
-// solution required
+
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-125-palindromic-sums.md:54]: stray blank line

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-125-palindromic-sums.md [hunk 2]
```diff
@@ -12,31 +12,75 @@ The palindromic number 595 is interesting because it can be written as the sum o
...
+          if (IsPalindrome(number) && !list.has(number)){
+             sum += number;
+             list.add(number);                        
+         }
+  
+     }
+ }
+   return sum;
+ }
+
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-125-palindromic-sums.md:85]: stray blank line

File: curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-125-palindromic-sums.md [hunk 3]
```diff
@@ -12,31 +12,75 @@ The palindromic number 595 is interesting because it can be written as the sum o
 
 There are exactly eleven palindromes below one-thousand that can be written as consecutive square sums, and the sum of these palindromes is 4164. Note that $1 = 0^2 + 1^2$ has not been included as this problem is concerned with the squares of positive integers.
 
-Find the sum of all the numbers less than $10^8$ that are both palindromic and can be written as the sum of consecutive squares.
+Find the sum of all the numbers less than the limit that are both palindromic and can be written as the sum of consecutive squares.
 
 # --hints--
 
-`palindromicSums()` should return `2906969179`.
+ `palindromicSums(100000000)` should return `2906969179`.
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-125-palindromic-sums.md:19]:
```suggestion
`palindromicSums(100000000)` should return `2906969179`.
```

**Comment by jeremylt** [comment]
@Florenciasicre are you still working on our review comments?

**Comment by jeremylt** [comment]
Cool, the big to-do left is updating the title and putting the checklist back!

**Review by gikf — COMMENTED** [review]
Generally it's looking good.

I've noticed couple details, see the specific comments below.

**Review by jeremylt — APPROVED** [review]
I think all the changes have been addressed and this is ready

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Pull request merged**